tests/integration/test_image_filters.py

```diff
@@ -120,6 +120,84 @@ def test_rgb_to_hed_raises_exception_on_gs_image():
     assert str(err.value) == "Input image must be RGB."
 
 
+def test_hematoxylin_channel_filter_with_rgb_image():
+    img = RGB.TCGA_LUNG_RGB
+    expected_value = load_expectation(
+        "pil-images-rgb/tcga-lung-rgb-hematoxylin-channel", type_="png"
+    )
+
+    hematoxylin_img = imf.hematoxylin_channel(img)
+
+    np.testing.assert_array_almost_equal(
+        np.array(hematoxylin_img), np.array(expected_value)
+    )
+    assert (
+        np.unique(np.array(ImageChops.difference(hematoxylin_img, expected_value)))[0]
+        == 0
+    )
+
+
+def test_hematoxylin_channel_filter_with_rgba_image():
+    img = RGBA.TCGA_LUNG
+    expected_value = load_expectation(
+        "pil-images-rgba/tcga-lung-hematoxylin-channel", type_="png"
+    )
+
+    hematoxylin_img = imf.hematoxylin_channel(img)
+
+    np.testing.assert_array_almost_equal(
+        np.array(hematoxylin_img), np.array(expected_value)
+    )
+    assert (
+        np.unique(np.array(ImageChops.difference(hematoxylin_img, expected_value)))[0]
+        == 0
+    )
+
+
+def test_hematoxylin_channel_raises_exception_on_gs_image():
+    gs_img = GS.DIAGNOSTIC_SLIDE_THUMB_GS
+
+    with pytest.raises(Exception) as err:
+        imf.hematoxylin_channel(gs_img)
+
+    assert isinstance(err.value, Exception)
```

Comment:
it would be better to check against `ValueError` since it's the one you explicitly raise